content/source/docs/enterprise/private/vmware-setup-guide.html.md

```diff
@@ -30,13 +30,12 @@ Depending on the chosen [operational
 mode](https://www.terraform.io/docs/enterprise/private/install-installer.html#operational-mode-decision),
 the infrastructure requirements for PTFE range from a single virtual machine
 for demo or proof of concept installations, to multiple virtual machines
-hosting postgres, access to S3 or Azure storage, and external Vault servers for
+hosting postgres, access to S3, Azure or Google Cloud storage, and external Vault servers for
```

Comment:
Wonder if we should go formal and use PostgreSQL here. I'm never sure about that.